### ISSUE TITLE
Installer error on macOS Ventura 13.4 (Build 22F66)

### ISSUE THREAD

**Issue by Qonfused** [Qonfused/OSX-Hyper-V#1]
macOS version:
```sh
$ uname -a
> Darwin iMac.mshome.net 22.5.0 Darwin Kernel Version 22.5.0: Mon Apr 24 20:51:50 PDT 2023; root:xnu-8796.121.2~5/RELEASE_X86_64 x86_64
```

Screenshot:
![image](https://github.com/Qonfused/OSX-Hyper-V/assets/32466081/8ad13f40-4e20-4ba8-b7c7-306016309488)

Installer log:
[Installer Log 13-Jun-2023.txt](https://github.com/Qonfused/OSX-Hyper-V/files/11740430/Installer.Log.13-Jun-2023.txt)
```console
13516 | Jun 14 00:41:34 iMac osishelperd_intel[550]: 8BE4DF61-93CA-11D2-AA0D-00E098032B8C:BootOrder = (null)
13517 | Jun 14 00:41:34 iMac osishelperd_intel[550]: 8BE4DF61-93CA-11D2-AA0D-00E098032B8C:BootNext = (null)
13519 | Jun 14 00:41:35 iMac osishelperd_intel[550]: -[OSISHelper blessUpdateBundleAtPath:mutableProductPath:enclosingMountPoint:withReply:] failed to copy /Volumes/macOS/macOS Install Data/UpdateBundle/AssetData/./boot/Firmware/System/Library/CoreServices/bootbase.efi
13520 | Jun 14 00:41:35 iMac osinstallersetupd[536]: -[OSISHelperProxy blessUpdateBundleAtPath:mutableProductPath:enclosingMountPoint:]_block_invoke_2: Error Domain=NSCocoaErrorDomain Code=260 "The file “bootbase.efi” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Volumes/macOS/macOS Install Data/UpdateBundle/AssetData/./boot/Firmware/System/Library/CoreServices/bootbase.efi, NSUnderlyingError=0x600001e6ce40 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
13521 | Jun 14 00:41:35 iMac osinstallersetupd[536]: Unable to setup bless on disk.
```

**Comment by Qonfused** [comment]
A workaround is to install Monterey and upgrade to Ventura within macOS (tested on 12.5.1, Build 21G83). Still investigating the cause on Ventura.

**Comment by Qonfused** [comment]
Appears to be resolved when testing Ventura 12.7, build 21G816. Don't have any more information to confirm if/when it was resolved on prior versions.

**Comment by v2p-kiev** [comment]
test
